src/engine/ruleset/integrations/windows/kvdbs/windows_eventid_to_message.json

```diff
@@ -0,0 +1,409 @@
+    "5888": "An object in the COM+ Catalog was modified.",
+    "5889": "An object was deleted from the COM+ Catalog.",
+    "5890": "An object was added to the COM+ Catalog.",
+    "6144": "Security policy in the group policy objects has been applied successfully.",
+    "6145": "One or more errors occurred while processing security policy in the group policy objects.",
+    "6272": "Network Policy Server granted access to a user.",
+    "6273": "Network Policy Server denied access to a user.",
+    "6274": "Network Policy Server discarded the request for a user.",
+    "6275": "Network Policy Server discarded the accounting request for a user.",
+    "6276": "Network Policy Server quarantined a user.",
+    "6277": "Network Policy Server granted access to a user but put it on probation because the host did not meet the defined health policy.",
+    "6278": "Network Policy Server granted full access to a user because the host met the defined health policy.",
+    "6279": "Network Policy Server locked the user account due to repeated failed authentication attempts.",
+    "6280": "Network Policy Server unlocked the user account.",
+    "6281": "Code Integrity determined that the page hashes of an image file are not valid. The file could be improperly signed without page hashes or corrupt due to unauthorized modification. The invalid hashes could indicate a potential disk device error",
+    "6400": "BranchCache: Received an incorrectly formatted response while discovering availability of content. ",
+    "6401": "BranchCache: Received invalid data from a peer. Data discarded. ",
+    "6402": "BranchCache: The message to the hosted cache offering it data is incorrectly formatted. ",
+    "6403": "BranchCache: The hosted cache sent an incorrectly formatted response to the client.",
+    "6404": "BranchCache: Hosted cache could not be authenticated using the provisioned SSL certificate. ",
+    "6405": "BranchCache: %2 instance(s) of event id %1 occurred.",
+    "6406": "%1 registered to Windows Firewall to control filtering for the following: %2",
+    "6407": "1%",
+    "6408": "Registered product %1 failed and Windows Firewall is now controlling the filtering for %2",
+    "6409": "BranchCache: A service connection point object could not be parsed.",
+    "6410": "Code integrity determined that a file does not meet the security requirements to load into a process.",
+    "6416": "A new external device was recognized by the System",
+    "6417": "The FIPS mode crypto selftests succeeded.",
+    "6418": "The FIPS mode crypto selftests failed.",
+    "6419": "A request was made to disable a device",
+    "6420": "A device was disabled.",
+    "6421": "A request was made to enable a device.",
+    "6422": "A device was enabled.",
+    "6423": "The installation of this device is forbidden by system policy",
+    "6424": "The installation of this device was allowed, after having previously been forbidden by policy."
+}
```

Comment:
Add a line break at the end of the file.

Reply:
Done